.github/workflows/test_installer.yml

```diff
@@ -0,0 +1,138 @@
+on: [push]
+
+jobs:
```

Comment:
nit: You could probably use a matrix strategy for the tests below.
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Great idea

Reply:
Done